src/api/schoolsMapInfo.ts

```diff
@@ -0,0 +1,72 @@
+import { gql, LazyQueryResult, useLazyQuery } from '@apollo/client';
```

Comment:
Why do we need to repeat this code (similar to file schoolsListing)? Maybe we could use one function instead of two?

Reply:
Removed that file, will create it with proper content in new PR